src/connector/src/sink/redis.rs

```diff
@@ -292,7 +292,7 @@ impl Sink for RedisSink {
                 .get(REDIS_VALUE_TYPE)
                 .map(|s| s.as_str())
             {
-                Some(REDIS_VALUE_TYPE_STRING) => {
+                Some(REDIS_VALUE_TYPE_STRING) | None => {
```

Comment:
Could you add some doc? This pattern - matching branch doesn't seem easy to understand.